app/components/primer/dropdown/menu_component.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Primer
+  module Dropdown
+    # DropdownMenus are lightweight context menus for housing navigation and actions.
+    # They're great for instances where you don't need the full power (and code)
+    # of the select menu.
```

Comment:
Do we need this description of we don't intend for this component to be consumed directly?